lib/generators/aws_record/base.rb

```diff
@@ -1,213 +0,0 @@
-# frozen_string_literal: true
-
-require 'rails/generators'
-require_relative 'generated_attribute'
-require_relative 'secondary_index'
-
-module AwsRecord
-  module Generators
-    class Base < Rails::Generators::NamedBase
```

Comment:
I think I'm slightly confused about this - Did this functionality move?

Reply:
Edit: oh nvm, I see in model_generator.rb

Reply:
I removed the base class because it doesn't make sense, and put all of the functionality inside the generator.